src/utils/renderMeasuringCells.tsx

```diff
@@ -0,0 +1,29 @@
+import { css } from '@linaria/core';
+import type { CalculatedColumn } from '../types';
+
+const measuringCellClassname = css`
+  @layer rdg.MeasuringCell {
```

Comment:
Does it need a layer? Does not hurt but wondering as we will never override the styles or it will conflict with other styles

Reply:
It probably doesn't. Doesn't hurt to add a layer anyway 🤷‍♂️